sdk/core/core-amqp/review/core-amqp.api.md

```diff
@@ -8,13 +8,13 @@ import { AbortSignalLike } from '@azure/abort-controller';
 import { AccessToken } from '@azure/core-auth';
 import { AmqpError } from 'rhea-promise';
 import { Message as AmqpMessage } from 'rhea-promise';
```

Comment:
Nit: Can we keep this as `Message` and not use the `Amqp` prefix here? Everywhere else where we use the `Amqp` prefix, we are referring to the type/interface defined in core-amqp which has camel casing. Where as this `AmqpMessage` is from rhea-promise. To be consistent with `MessageHeader` and `MessageProperties` from rhea-promise, this should just `Message` as well

Reply:
We either do that or use the `Rhea` prefix

Reply:
I was thinking it, but refrained from doing since it would change service-bus and event-hubs.
But it is probably worth doing it.. will do!